.github/workflows/docker.yml

```diff
@@ -0,0 +1,49 @@
+name: docker
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      # build for every branch push
+      - "**"
+    tags:
+      # Build for semver formatted tags
+      - "v*.*.*"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
```

Comment:
In the [`package.json`](https://github.com/wisdom-oss/frontend/blob/f76c3df98faa87f03e629b0457f57e6f4892e274/package.json#L16) I used `wisdom-oss/frontend` as build tag. Maybe we should do this here too, otherwise the name is very unclear. Also do we want to add a version number or branch name?

Reply:
That will be the same name, was `${{ github.repository }}` will evaluate into `wisdom-oss/frontend` as documented here. This allows for moving the image and creating a new package to keep better track of where the image originated from.

The version numbers and/or the branch name are added here: https://github.com/wisdom-oss/actions/blob/main/docker-build/multiplatform/merge/action.yml